src/airflow_docker/ext/python_callable.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#
+#     Copyright 2019 Contributing Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import base64
+import json
+import textwrap
+
+from airflow.utils.db import provide_session
+
+from airflow_docker.constants import CONTAINER_RUN_DIR
+from airflow_docker.context import serialize_context
+
+PYTHON_ENTRYPOINT = textwrap.dedent(
+    """\
+#!/usr/bin/env python
```

Comment:
should this be a separate file that you `open` instead?

Reply:
🤷‍♂ I kind of like it here since it's a python string template (and it's completely tied to the implementation in this module). But if you felt strongly I'd move it out.

Reply:
I was mostly thinking for like linting/highlighting/etc. otherwise i dont really feel strongly.